packages/core/deploy/006_deploy_voting.js

```diff
@@ -0,0 +1,41 @@
+const { ZERO_ADDRESS } = require("@uma/common");
+
+const func = async function (hre) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+
+  const { deployer } = await getNamedAccounts();
+
+  const Timer = (await deployments.getOrNull("Timer")) || { address: ZERO_ADDRESS };
+  const VotingToken = await deployments.get("VotingToken");
+  const Finder = await deployments.get("Finder");
+
+  // Set the GAT percentage to 5%
+  const gatPercentage = { rawValue: web3.utils.toWei("0.05", "ether") };
+
+  // Set the inflation rate.
+  const inflationRate = { rawValue: web3.utils.toWei("0.0005", "ether") };
+
+  // Set the rewards expiration timeout.
+  const rewardsExpirationTimeout = 60 * 60 * 24 * 14; // Two weeks.
+
+  // Set phase length to one day.
+  const phaseLength = "86400";
```

Comment:
What could be kind of useful is detecting whether we are on a production network or not, and setting phase length to 10 mins or shorter if not!

Reply:
Interesting..... I might add that

Reply:
As discussed offline, I think we should wait on a follow-up for this feature.